website/static/css/pages/home-page.css

```diff
@@ -25,9 +25,6 @@ h3, h4 {
     background-size: cover;
     color: #fff;
 }
-.meetings .btn {
```

Comment:
Nope, pretty sure it was just a duplicate of [this styling](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/home-page/meetingsAndConferencesPlugin.js#L25).